app/controllers/departments_controller.rb

```diff
@@ -1,5 +1,6 @@
 class DepartmentsController < ApplicationController
   before_action :set_department, only: [:show, :edit, :update, :destroy]
+  before_action :admin_employee
```

Comment:
Your actions should read like actions. You want your syntax here to read well. What is the method admin_employee doing? Can that be somewhat inferred without having to dig through the method itself? You can accomplish that by using something like "verify_admin_employee" versus "admin_employee"